code-of-conduct-ID.md

```diff
@@ -0,0 +1,40 @@
+## Pengantar
+
+Kami berdedikasi untuk memberikan pengalaman bebas pelecehan untuk semua orang, dan kami tidak mentolerir pelecehan peserta dalam bentuk apapun. Kami meminta Anda untuk memperhatikan orang lain dan bersikap profesional serta hormat kepada peserta lainnya. Kode dan prosedur terkait ini juga berlaku untuk perilaku yang tidak dapat diterima yang terjadi di luar lingkup kegiatan komunitas, di semua tempat komunitas— daring dan pribadi— serta komunikasi secara pribadi, dan dimanapun perilaku yang memiliki potensi untuk mempengaruhi keselamatan serta kesejahteraan anggota komunitas secara negatif. Peserta pameran, pembicara, sponsor, staf dan semua peserta lainnya pada acara yang diselenggarakan oleh Docker, Inc (DockerCon, *meetup*, *user group*) atau yang diadakan di fasilitas Docker, Inc tunduk pada Pedoman Komunitas dan Kode Etik ini.
```

Comment:
* On `1` should `menoleransi` for formal form in Bahasa Indonesia, can be checked to https://kbbi.kemdikbud.go.id/entri/menoleransi . `Tolerir` is non formal form in Bahasa Indonesia.

* On `4` and `6`, I think no need to changed.